spec/features/reg_users/order_show_page_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User Order Show Page' do
+  describe 'when a registered user visits their profile orders page' do
+    describe 'they click on an order id number' do
+      it 'lets them see that order show page' do
+        @user = create(:user)
+        @order_1 = create(:order, user: @user)
+
+        @item_1 = create(:item)
+        @item_2 = create(:item)
+
+        @order_item_1= create(:fulfilled_order_item, item: @item_1, order: @order_1)
+        @order_item_2= create(:fulfilled_order_item, item: @item_2, order: @order_1, quantity: 3)
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
+        visit profile_orders_path
+        click_on "#{@order_1.id}"
```

Comment:
Just for reference, you should just be able to put in only @order_t.id.to_s if that is easier to understand in the future